framework/helpers/BaseConsole.php

```diff
@@ -340,6 +340,93 @@ public static function ansiStrlen($string)
         return mb_strlen(static::stripAnsiFormat($string));
     }
 
+    //
+    // Error code constant names, values, and descriptions are taken from
+    // OpenBSD's sysexits(3) man page http://man.openbsd.org/sysexits
```

Comment:
Good question.

Afaik, this is the only thing like a standard for exit codes. It's been around for a long time, is fairly well-know if not widely used, and it's been adopted far beyond the BSD world. Some popular Linux distros come with it in `/usr/include/sysexits.h` (if not in the manual). 

Also check out https://github.com/search?q=sysexits

I think it's clear that these code definitions are not platform specific. What's less clear is whether many people think that just using `0` and `1` is insufficient.